ui/components/DashboardComponent.js

```diff
@@ -222,6 +232,10 @@ class DashboardComponent extends React.Component {
     this._isMounted = true
     this.fetchAvailableAdapters();
 
+    fetchAllContexts(25)
+      .then(res => this.setState({ contexts : res.contexts }))
```

Comment:
probably add an extra check if contexts field is present in the res. We had few problems in the past handling undefined in such dynamic fetch calls.

Reply:
Assuming `contexts` is an array

```suggestion
      .then(res => this.setState({ contexts : res?.contexts || [] }))
```